WPFHospitalEditor/Controller/Interface/IExaminationServerController.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace WPFHospitalEditor.Controller.Interface
+{
+    public interface IExaminationServerController
+    {
+        public string ScheduleExamination(DateTime startTime, int DoctorId, int patientId);
+    }
```

Comment:
Public access modifier here is not needed, same mistake in IExaminationServerService.cs

Reply:
fixed